From 446b6f015da0e47128f3563e151efb16cdc29c52 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 8 Feb 2021 22:34:33 +0000
Subject: IWYU: include memory for std::unique_ptr

Bug: None
Change-Id: I2a4bd67ad74106eb145389c1320d34229e514bb2
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2679393
Reviewed-by: Christoph Schwering <schwering@google.com>
Commit-Queue: Christoph Schwering <schwering@google.com>
Cr-Commit-Position: refs/heads/master@{#851928}
---
 components/autofill/core/browser/data_model/contact_info.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/autofill/core/browser/data_model/contact_info.h b/components/autofill/core/browser/data_model/contact_info.h
index 275a977f148e..f0517aa95e37 100644
--- a/components/autofill/core/browser/data_model/contact_info.h
+++ b/components/autofill/core/browser/data_model/contact_info.h
@@ -5,6 +5,7 @@
 #ifndef COMPONENTS_AUTOFILL_CORE_BROWSER_DATA_MODEL_CONTACT_INFO_H_
 #define COMPONENTS_AUTOFILL_CORE_BROWSER_DATA_MODEL_CONTACT_INFO_H_
 
+#include <memory>
 #include <string>
 #include <vector>
 
-- 
2.25.1

